src/main.py

```diff
@@ -24,3 +25,12 @@ async def readyz() -> dict:
 async def healthz() -> dict:
     """The endpoint for the Health Probe."""
     return {"healthy": "true"}
+
+
+if __name__ == "__main__":
+    uvicorn.run(
+        "main:app",
+        host="localhost",
+        port=8000,
+        reload=True,
+    )
```

Comment:
Actually your first change was correct, we don't need reload because we can provide `--reload` with IDEs (PyCharm, VS Code). The reason why we disable reload is that prod should have have auto-reload. 
just small improvements. Can you apply the following? 
```suggestion
if __name__ == "__main__":
    uvicorn.run(
        app,
        host="0.0.0.0",
        port=8000,
    )
```

Reply:
Please document that we should now provide auto reload with with IDEs. 
There is --reload argument.